### PR TITLE
[Release 1.12.0] Add release logs for 1.12.0rc1

### PR DESCRIPTION
**Pull request by jianoaix**
## Why are these changes needed?

Add release logs for 1.12.0rc1. The base is 1.11.0rc1.

## Related issue number

N/A

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/release_logs/1.12.0/benchmarks/many_pgs.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "_peak_memory": 3.85,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n207\t0.26GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n182\t0.17GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n1927\t0.15GiB\tpython distributed/test_many_tasks.py --num-tasks=1000\n283\t0.13GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n69\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n441\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n1973\t0.06GiB\tray::MemoryMonitorActor.run()\n70\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=31a3d4a5\n398\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=31a3d4a5\n200\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.71.106:9031 --host=0.0.0.0",
+    "num_tasks": 1000,
+    "success": "1",
+    "tasks_per_second": 7.071159188773613,
```
Comment (scv119) [inline release/release_logs/1.12.0/benchmarks/many_pgs.json:6]: this looks like regression. It was 17 in 1.11.0 https://github.com/ray-project/ray/blob/master/release/release_logs/1.11.0/benchmarks/many_pgs.json

File: release/release_logs/1.12.0/microbenchmark.json [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    "client__1_1_actor_calls_concurrent": [
+        930.4997845873929,
+        10.420162071443412
+    ],
+    "client__1_1_actor_calls_sync": [
+        535.3811503308413,
+        3.02536760907699
+    ],
+    "client__get_calls": [
+        1105.2486940332228,
```
Comment (scv119) [inline release/release_logs/1.12.0/microbenchmark.json:47]: this was 1584

**Comment by jianoaix** [comment]
Sang is OOO, adding Chen instead.

File: release/release_logs/1.12.0/scalability/object_store.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (scv119) [inline release/release_logs/1.12.0/scalability/object_store.json:1]: looks good

File: release/release_logs/1.12.0/benchmarks/many_nodes.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
```
[the remark this answers is not in the document]
Comment (jianoaix) [reply release/release_logs/1.12.0/benchmarks/many_nodes.json:1]: tasks_per_second seems to have a significant improvement, if it's not noise.

File: release/release_logs/1.12.0/benchmarks/many_pgs.json (hunk 1 above)
Comment (jianoaix) [reply release/release_logs/1.12.0/benchmarks/many_pgs.json:6]: Also the time was 58 but now 441

**Comment by jianoaix** [comment]
Btw, this PR intentionally had 2 commits, with the first one identical to 1.11.0rc1, so it should be a little easier to look at those numbers side by side here.

File: release/release_logs/1.12.0/benchmarks/many_pgs.json (hunk 1 above)
Comment (scv119) [reply release/release_logs/1.12.0/benchmarks/many_pgs.json:6]: false alarm. this is not the result of many_pgs.

File: release/release_logs/1.12.0/benchmarks/many_nodes.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    "_peak_memory": 3.85,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n207\t0.26GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n182\t0.17GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n1927\t0.15GiB\tpython distributed/test_many_tasks.py --num-tasks=1000\n283\t0.13GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n69\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n441\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n1973\t0.06GiB\tray::MemoryMonitorActor.run()\n70\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=31a3d4a5\n398\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=31a3d4a5\n200\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.71.106:9031 --host=0.0.0.0",
```
Comment (scv119) [inline release/release_logs/1.12.0/benchmarks/many_nodes.json:3]: @jianoaix  this looks a wrong copy as well.

File: release/release_logs/1.12.0/benchmarks/many_pgs.json (hunk 1 above)
Comment (jianoaix) [reply release/release_logs/1.12.0/benchmarks/many_pgs.json:6]: As it turned out, this is a false alarm \o/

The many_pgs.json is copying the results of above many_nodes (my mistake in manually copying files around), the real results for many_pgs.json looks good!

Comment (jianoaix) [reply release/release_logs/1.12.0/benchmarks/many_pgs.json]: Fixed.

File: release/release_logs/1.12.0/benchmarks/many_nodes.json (hunk 5 above)
Comment (jianoaix) [reply release/release_logs/1.12.0/benchmarks/many_nodes.json:3]: many_nodes test: https://buildkite.com/ray-project/release-tests-branch/builds/273#d0bb494a-8d06-41bf-ba40-66c406f82a57
It seems right.

**Review by jianoaix — COMMENTED** [review]
Thanks for careful eyeballing!

**Comment by mwtian** [comment]
If the changes are minor, we don't need to rerun the tests.

**Review by mwtian — APPROVED** [review]

**Comment by jianoaix** [comment]
Ping again for review or merge? Thanks.

**Comment by mwtian** [comment]
@scv119 @ericl ^

**Pull request merged**